<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Q3DSurface三维地形图</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_setupChart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>364</width>
      <height>603</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>旋转和平移</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>预设视角</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_preView">
            <item>
             <property name="text">
              <string>FrontLow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Front</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FrontHigh</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LeftLow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Left</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LeftHigh</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RightLow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Right</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RightHigh</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BehindLow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Behind</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BehindHigh</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sometricLeft</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sometricLeftHigh</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sometricRight</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sometricRightHigh</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DirectlyAbove</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DirectlyAboveCW45</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DirectlyAboveCCW45</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FrontBelow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LeftBelow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RightBelow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BehindBelow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DirectlyBelow</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>水平旋转</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_horizontalRotate">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>垂直旋转</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_VerticalRotate">
                <property name="minimum">
                 <number>-180</number>
                </property>
                <property name="maximum">
                 <number>180</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>缩放</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_scaled">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QToolButton" name="toolButton_up">
               <property name="text">
                <string>...</string>
               </property>
               <property name="arrowType">
                <enum>Qt::UpArrow</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QToolButton" name="toolButton_left">
               <property name="text">
                <string>...</string>
               </property>
               <property name="arrowType">
                <enum>Qt::LeftArrow</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QToolButton" name="toolButton_refresh">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QToolButton" name="toolButton_right">
               <property name="text">
                <string>...</string>
               </property>
               <property name="arrowType">
                <enum>Qt::RightArrow</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QToolButton" name="toolButton_down">
               <property name="text">
                <string>...</string>
               </property>
               <property name="arrowType">
                <enum>Qt::DownArrow</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>图形总体</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>图形主题</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_mapTheme">
            <item>
             <property name="text">
              <string>ThemeQt</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ThemePrimaryColors</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ThemeDigia</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ThemeStoneMoss</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ThemeArmyBlue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ThemeRetro</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ThemeEbony</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ThemeIsabelle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ThemeUserDefined</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>字体大小</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_fontSize"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>选择模式</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_selectMode">
            <item>
             <property name="text">
              <string>SelectionNone</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SelectionItem</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SelectionRow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SelectionItemAndRow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SelectionColumn</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SelectionItemAndColumn</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SelectionRowAndColumn</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SelectionItemRowAndColumn</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SelectionSlice</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SelectionMultiSeries</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="checkBox_showBackground">
            <property name="text">
             <string>显示背景</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_showGrid">
            <property name="text">
             <string>显示背景网格</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="checkBox_showReflection">
            <property name="text">
             <string>显示阴影</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_reverseValueAxis">
            <property name="text">
             <string>垂直坐标轴反向</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="checkBox_showAxisTitle">
            <property name="text">
             <string>显示轴标题</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_showAxisLabelBackground">
            <property name="text">
             <string>显示轴标签背景</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>序列设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>曲面样式</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_surfaceStyle">
            <item>
             <property name="text">
              <string>DrawWireframe</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DrawSurface</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DrawSurfaceAndWireframe</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,1,2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>单点形状</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_pointShape">
            <item>
             <property name="text">
              <string>MeshUserDefined</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshBar</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshCube</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshPyramid</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshCone</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshBevelBar</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshBevelCube</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshSphere</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshMinimal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshArrow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MeshPoint</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_smoothEffect">
            <property name="text">
             <string>单点光滑效果</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_showSelectedBarLabels">
            <property name="text">
             <string>显示半单点标签</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>平面着色</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_seriesColor"/>
   <addaction name="action_addRow"/>
   <addaction name="action_insertRow"/>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_seriesColor">
   <property name="text">
    <string>曲面颜色</string>
   </property>
   <property name="toolTip">
    <string>曲面颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_reBuildChart">
   <property name="text">
    <string>重新生成</string>
   </property>
   <property name="toolTip">
    <string>重新生成</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_changValue">
   <property name="text">
    <string>修改点坐标</string>
   </property>
   <property name="toolTip">
    <string>修改点坐标</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_addRow">
   <property name="text">
    <string>渐变色一</string>
   </property>
   <property name="toolTip">
    <string>渐变色一</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_insertRow">
   <property name="text">
    <string>渐变色二</string>
   </property>
   <property name="toolTip">
    <string>渐变色二</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_delRow">
   <property name="text">
    <string>删除行</string>
   </property>
   <property name="toolTip">
    <string>删除行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
